### ISSUE TITLE
Trouble reproducing `PTC` results from paper

### ISSUE THREAD

**Issue by bkj** [MLDroid/graph2vec_tf#4]
I'm running the `ptc` experiment using the command from `all_dataset_runner.sh`:
```
python main.py -c ../data/kdd_datasets/ptc -l ../data/kdd_datasets/ptc.Labels -b 256 -d 512 --wlk_h 3 -e 1000 -lr 0.5
```

However, I've run a few times, and I haven't really gotten results close to the 60.17 ± 6.86 reported in the paper.  I'm getting something in the mid to low 0.50s -- are these the right parameter settings?

Thanks
~ Ben

**Comment by annamalai-nr** [comment]
The results reported in the paper are based on a version of graph2vec which uses gensim not tensorflow (similar to [subgraph2vec_gensim](https://github.com/MLDroid/subgraph2vec_gensim) ) . Besides, the results heavily depend on hyperparameters such as --wlk_h, --epochs, --embedding_size and -neg. Please try larger values for the first three and smaller value for the last hyperparameter (say, -neg = 2 or 3). Different hyperparameter settings will help to improve the results. Besides, pls tune the learning rate and try replacing the optimizer (Adam or Adagrad may be more suitable).
